Use consistent language + script naming convention

Paul-Dempsey opened the pull request:
The Detect languages table is apparently showing Translator system-internal names with underscores, rather than the actual language names.  The edit changes these to proper language names according to the convention. this clarifies the ambiguity in "Haitian_Creole" -- where "Creole" is part of the name, not a script as it is in other identifiers.  I would ask that for consistency, you also show the language codes that the Detect API returns for all of these. I assume that the API returns proper ISO codes, and not these internal ids with underscores.

azurecla commented:
Hi __@stalker314314__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (branko). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Ja-Dunn commented:
@stalker314314 - Hi, thanks for your contribution. We are in the process of refactoring the Azure technical articles, so we can’t merge this PR. When the refactoring is complete, we can give you a link to the new location of the article you updated so you can resubmit your update, if you want to.

ktoliver commented:
@stalker314314 - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://docs.microsoft.com/en-us/azure/documentdb/documentdb-cpp-get-started](https://docs.microsoft.com/en-us/azure/documentdb/documentdb-cpp-get-started). Thanks for your understanding.

Commit:
Use consistent language + script naming convention

The Detect languages table is apparently showing Translator system-internal names with underscores, rather than the actual language names.  The edit changes these to proper language names according to the convention. this clarifies the ambiguity in "Haitian_Creole" -- where "Creole" is part of the name, not a script as it is in other identifiers.  I would ask that for consistency, you also show the language codes that the Detect API returns for all of these. I assume that the API returns proper ISO codes, and not these internal ids with underscores.
